src/css/components/navigation.less

```diff
@@ -26,6 +26,7 @@
 .navbar-inverse .navbar-nav > .active > a:focus {
   color: @black-haze;
   background-color: transparent;
+  cursor: pointer;
```

Comment:
If something is clickable we should provide the default feedback behavior for it (pointer). 

@pierlo-upitup not sure what Bootstrap docs you're referencing? Seems like active would still have pointer according to [this](https://getbootstrap.com/examples/navbar/#).